<?xml version="1.0" ?>
<robot name="object_name">
  <link name="baseLink">
    <contact>
    	<friction_anchor/>
    	   <lateral_friction value="1.0"/>
    </contact>
    <inertial>
      <origin rpy="0 0 0" xyz="0 0 0"/>
       <!-- mass controls penetration -->
       <mass value=".01"/>
       <inertia ixx="1e-4" ixy="0" ixz="0" iyy="1e-4" iyz="0" izz="1e-4"/>
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="object_name.obj"/>
      </geometry>
       <material name="white">
        <color rgba="1 1 1 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="object_name.obj"/>
      </geometry>
    </collision>
  </link>
</robot>



